test/gflags_unittest.cc

```diff
@@ -601,7 +601,7 @@ TEST(SetFlagValueTest, IllegalValues) {
             SetCommandLineOption("test_bool", "12"));
 
   EXPECT_EQ("",
-            SetCommandLineOption("test_uint32", "1970"));
+            SetCommandLineOption("test_uint32", "-1970"));
```

Comment:
Not sure why 1970 should be illegal value for uint32.
With my fix to typename it can hold this value.